core/layouts/geary/layout--geary.tpl.php

```diff
@@ -31,82 +31,92 @@
 
   <?php if ($content['header']): ?>
     <header class="l-header" role="banner" aria-label="<?php print t('Site header'); ?>">
-      <div class="container container-fluid">
-        <div class="row">
-          <?php print $content['header']; ?>
-        </div><!-- /.row -->
+      <div class="l-header-inner container container-fluid">
```

Comment:
To support color banding we need to have a wrapper with no bootstrap classes and an inner with container on *every* horizontal band of our layouts.

HTML bloaty... but will support more design choices without people having to touch markup.